main.go

```diff
@@ -39,7 +39,12 @@ func main() {
 
 	r := gin.Default()
 
-	serveStaticResources(staticPath, r)
+	if staticPath != "" {
+		r.StaticFS(staticPath, http.Dir(staticPath))
```

Comment:
Maybe better:
```
r.StaticFS("/static", http.Dir(staticPath))
r.GET("/", func(c *gin.Context) {
	c.Redirect(http.StatusMovedPermanently, "/static")
})
```

Reply:
Well, it's a value that may be visible for users. I would say that `/cloud-shell` is nicer to see here.
But I got your concern that this parameter has another assignment. Let me propose an alternative solution

Reply:
I've reverted the changing path for serving static content.
I have an idea to rework --static parameter to have format something like `/home/user/frontend:/cloud-shell` where the first part before : is absolute folder name and the second one REST path fragment to access that resources.
But it would require some changes I would like to propose as a separate PR when I have time.